prototype_source/graph_mode_dynamic_bert_tutorial.rst

```diff
@@ -0,0 +1,551 @@
+(prototype) Graph Mode Dynamic Quantization on BERT
+==============================================
+
+
+**Author**: `Supriya Rao <https://github.com/supriyar>`_
+
+Introduction
+------------
+
+This tutorial introduces the steps to do post training Dynamic Quantization with Graph Mode Quantization. Dynamic quantization converts a float model to a quantized model with static int8 data types for the weights and dynamic quantization for the activations. The activations are quantized dynamically (per batch) to int8 while the weights are statically quantized to int8. Graph Mode Quantization flow operates on the model graph and requires minimal user intervention to quantize the model. To be able to use graph mode, the float model needs to be either traced or scripted first.
```

Comment:
Is there documentation/doc stings or source code we can link to throughout this tutorial or at least in the introduction for users to know where to find additional info?

Reply:
Sure, how do I link to something that hasn't been released yet? For example `:func:`~torch.quantization.quantize_dynamic_jit` in https://github.com/pytorch/pytorch/blob/master/docs/source/quantization.rst#id153

Reply:
Could you link to the sub-header link for now?
https://pytorch.org/docs/master/quantization.html#torch-quantization

or could you link to the source code? 
https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantize_jit.py